app/controllers/request-to-go-live/go-live-stage-to-next-page-path.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+// Local dependencies
+const goLiveStage = require('../../models/go-live-stage')
+const paths = require('../../paths')
```

Comment:
could make this ```paths = require('../../paths').requestToGoLive``` and remove the `requestToGoLive` each time further down.

Reply:
Thanks, I changed that to use `Destructuring assignment`